base/base/ReplxxLineReader.cpp

```diff
@@ -220,6 +220,35 @@ ReplxxLineReader::ReplxxLineReader(
     rx.bind_key(Replxx::KEY::control('W'), [this](char32_t code) { return rx.invoke(Replxx::ACTION::KILL_TO_WHITESPACE_ON_LEFT, code); });
 
     rx.bind_key(Replxx::KEY::meta('E'), [this](char32_t) { openEditor(); return Replxx::ACTION_RESULT::CONTINUE; });
+
+    /// readline insert-comment
+    auto insert_comment_action = [this](char32_t code)
+    {
+        replxx::Replxx::State state(rx.get_state());
+        const char * line = state.text();
+        const char * line_end = line + strlen(line);
+
+        std::string commented_line;
+        if (std::find(line, line_end, '\n') != line_end)
```

Comment:
Don't we also want to uncomment by shortcut if query was commented?

Reply:
There is no such bindings in readline:

```
$ bind -p | grep comment
"\e#": insert-comment
```

Do you think it really worth it? Maybe it does not exists for a reason, for example not always you will get back to the commented query/command shortly.